File: TelerikUIExamplesInCSharp/Infrastructure/OptionSection.cs
using System;
using System.Collections.Generic;

namespace Examples
{
	public class OptionSection
	{
		public string Title {
			get;
			set;
		}

		public List<OptionInfo> Items {
			get;
			set;
		}

		public int SelectedOption {
			get;
			set;
		}

		public OptionSection (string title)
		{
			this.Title = title;
			this.Items = new List<OptionInfo> ();
		}
	}
}

File: TelerikUIExamplesInSwift2.0/TelerikUIExamplesInSwift/OptionSection.swift
//
//  OptionSection.swift
//  TelerikUIExamplesInSwift
//
//  Copyright (c) 2015 Telerik. All rights reserved.
//

import UIKit

class OptionSection: NSObject {
   
    var title = ""
    
    var items = NSMutableArray()
    
    var selectedOption: Int = 0

    init(Text:String) {
        self.title = Text
    }
}
